components/assetstore-controller-manager/Makefile

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
The Makefile has to be adjusted. And we probably need `before-commit.sh`, don't we?

Reply:
Okay then, so I won't touch it here 😄